### PR TITLE
Mad Documentation, Yo'

### PR DESCRIPTION
**Pull request by jcrussell**
Cleaned up the HelpShort/HelpLong for the various CLI handlers. Made some minor code cleanups as I was reading through.

### CODE REVIEW

File: src/minimega/deploy.go [hunk 1]
```diff
@@ -20,23 +20,29 @@ import (
 var deployCLIHandlers = []minicli.Handler{
 	{ // deploy
 		HelpShort: "copy and run minimega on remote nodes",
-		HelpLong: `deploy will copy and run minimega on remote nodes,
-facilitating the deployment of minimega on a cluster. By default, deploy will
-launch minimega with the same flags used when starting this minimega, and add
-the -nostdin flag so that the remote minimega can be backgrounded.
+		HelpLong: `
+Deploy copies and runs minimega on remote nodes, facilitating the deployment of
```
Comment (djfritz) [inline src/minimega/deploy.go]: I've been in the habit of using API names as special nouns of a sort, where the API is explicitly not capitalized. We do that explicitly for 'minimega' as well on the website, even when it's the beginning of a sentence. 

Thoughts on that practice? If it's agreeable, we should change Deploy to deploy.

File: src/minimega/deploy.go [hunk 2]
```diff
@@ -20,23 +20,29 @@ import (
...
+minimega to a cluster. By default, deploy will launch minimega with the same
+flags used when starting this minimega, and add the -nostdin flag so that the
+remote minimega can be backgrounded. For example, to launch minimega on nodes
+kn1 and kn2:
+
+	deploy launch kn[1-2]
 
-deploy uses scp/ssh to copy and run minimega. By default, minimega will attempt
+Deploy uses scp/ssh to copy and run minimega. By default, minimega will attempt
```
Comment (djfritz) [inline src/minimega/deploy.go]: Same as previous comment.

File: src/minimega/dot.go [hunk 3]
```diff
@@ -56,11 +55,8 @@ func cliDot(c *minicli.Command) *minicli.Response {
 	}
 	defer fout.Close()
 
-	cmd, err := minicli.CompileCommand(".columns host,name,id,ip,ip6,state,vlan vm info")
-	if err != nil {
-		// Should never happen
-		log.Fatalln(err)
-	}
+	// TODO: Rewrite to use runCommandGlobally
+	cmd := minicli.MustCompile(".columns host,name,id,ip,ip6,state,vlan vm info")
```
Comment (djfritz) [inline src/minimega/dot.go]: Much better now that we have this.

File: src/minimega/history.go [hunk 4]
```diff
@@ -12,6 +12,14 @@ import (
 var historyCLIHandlers = []minicli.Handler{
 	{ // history
 		HelpShort: "show command history",
+		HelpLong: `
+History displays a list of all the commands that have been invoked since
```
Comment (djfritz) [inline src/minimega/history.go]: Same case comment as above.

File: src/minimega/misc_cli.go [hunk 5]
```diff
@@ -22,10 +22,10 @@ var miscCLIHandlers = []minicli.Handler{
 	{ // quit
 		HelpShort: "quit minimega",
 		HelpLong: `
-Quit. An optional integer argument X allows deferring the quit call for X
-seconds. This is useful for telling a mesh of minimega nodes to quit.
+Quit minimega. An optional integer argument X allows deferring the quit call
+for X seconds. This is useful for telling a mesh of minimega nodes to quit.
 
-quit will not return a response to the cli, control socket, or meshage, it will
+Quit will not return a response to the cli, control socket, or meshage, it will
```
Comment (djfritz) [inline src/minimega/misc_cli.go]: case

**Comment by djfritz** [comment]
Testing handler changes now...

**Comment by jcrussell** [comment]
I'm fine with changing the case to lowercase everywhere, as long as it's consistent.

File: src/minimega/optimize.go [hunk 6]
```diff
@@ -83,7 +83,9 @@ To disable all optimizations see "clear optimize".`,
 Resets state for virtualization optimizations. See "help optimize" for more
 information.`,
 		Patterns: []string{
-			"clear optimize [affinity,]",
+			"clear optimize",
```
Comment (djfritz) [inline src/minimega/optimize.go]: we're missing 'clear optimize ksm'

**Comment by djfritz** [comment]
Updated patterns all work.

**Comment by djfritz** [comment]
Roger. Push another commit fixing the case and the LGTM.

**Comment by djfritz** [comment]
Oh, and the 'clear optimize ksm' - it has a default value of false, we should support clearing it.

**Comment by jcrussell** [comment]
Think it's ready to merge...

**Comment by djfritz** [comment]
LGTM

**Pull request merged**